low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -197,6 +199,33 @@ SYSTEMDWATCHDOG:
    smart_test_interval: 999999999
    run_smart_on_start: 'False'
 
+SERVICEMONITOR:
+   monitor: true
+   threaded: true
+   thread_sleep: 1
+   polling_frequency: 15
+   threshold_inactive_time: 60
+   monitored_services:
+      - rabbitmq-server.service
+      - hare-consul-agent.service
+      - hare-consul-agent-c1.service
+      - hare-consul-agent-c2.service
+      - elasticsearch.service
+      - kibana.service
```

Comment:
Remove kibana, corosync, pacemaker, pcsd service from list and add "multipathd.service".